### DISCUSSION TITLE
not a bug - reported crash but problem is my laptop

### DISCUSSION THREAD

**Discussion by kerednekt1a** [rrirower/Channel-Feed-Manager#206] [General]
I installed Channel Feed Manager on my backup laptop but although the app launched I got immediately an error and gave feedback.

The problem is my backup laptop where the user is installed on drive D not C.

This has not been a problem before because I only use my backup  for emails.

Apologies if you are getting these crash reports.

**Comment by rrirower** [comment]
No worries.  Please make sure to fill out the crash screen info if it displays.

**Comment by urapps** [comment]
Re my previous problem I just submitted another crash report regarding the same problem.
I discovered the problem was caused by 2 documents folders existing - one on C drive and one on D drive.
I ran the installer from my desktop (instead of download folder) and installation ran ok and now program runs without crashing.
Please ignore my latest crash report.
Thanks for a great program!
